src/plugins/image-upload/event-handlers.ts

```diff
@@ -0,0 +1,11 @@
+window.document.addEventListener(
+  'dragover',
+  event => (
+    event.preventDefault(),
+    !(<HTMLElement>event?.target).closest('.ProseMirror') &&
+      event.dataTransfer !== null &&
+      (event.dataTransfer.dropEffect = 'none')
+  )
+)
+
+export {}
```

Comment:
No removeEventListener somewhere?

Reply:
Yes, I decided it was not really worth it because the listener will always be needed unless the user leaves the application.